### DISCUSSION TITLE
Icons with relative path

### DISCUSSION THREAD

**Discussion by martina-oefelein** [likec4/likec4#939] [Q&A] [answered]
Is it possible to refer icons using a relative path, so that I can store custom icons in the same repository as the c4 files?

```c4
    icon ../icons/hive.svg
```

When I try this, no error is shown, but the icon is not shown

(I'm using the preview in VS Code.)

**Comment by davydkov** [comment, marked as answer]
Hi @martina-oefelein

At the moment likec4 only allows [bundled icons](https://likec4.dev/dsl/styling/#bundled-icons) or publicly available resources.

If it possible in your case, I would suggest to upload even to github gists
